CLAUDE.md

```diff
@@ -2,69 +2,35 @@
 
 This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
 
-## Project Overview
-
-Liam ERD is a database schema visualization tool that generates beautiful, interactive ER diagrams. The project consists of a web application, CLI tool, and multiple supporting packages in a pnpm monorepo structure.
-
 ## Essential Commands
 
-### Development
-```bash
-# Start development servers
-pnpm dev
-
-# Build all packages
-pnpm build
+- pnpm dev - Start dev servers
+- cd frontend/apps/app && pnpm dev - Start dev server for specific package
+- pnpm build - Build all packages
+- pnpm lint - Run linting and formatting
+- pnpm test - Run tests
+- pnpm fmt - Run format code
 
-# Run linting and formatting
-pnpm lint
-pnpm fmt
+### App-specific Commands
 
-# Run tests
-pnpm test
-pnpm test:e2e
-
-# Generate code and CSS type definitions
-pnpm gen:turbo
-```
-
-### App-specific Development
-```bash
+````bash
 # Run only the main web app (port 3001)
-cd frontend/apps/app && pnpm dev
-
-# Run with CSS generation
-cd frontend/apps/app && pnpm dev:css
+pnpm --filter @liam-hq/app dev
```

Comment:
Personally, I prefer running with `--filter` over running after `cd`. When humans copy and run commands, `cd` requires going back.